docs/man_pages/project/creation/init.md

```diff
@@ -0,0 +1,20 @@
+init
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns init [--path <Directory>] [--force]`
+
+Initializes a project for development. It will ask you interactively to configure the project and will write a package.json for you.
```

Comment:
Initializes a project for development. The command prompts you to provide your project configuration interactively and uses the information to create a new `package.json` file or update the existing one.